packages/edge/package.json

```diff
@@ -8,17 +8,17 @@
   "scripts": {
     "build": "tsup src/index.ts --dts --format esm,cjs",
     "test": "jest --env node --verbose --runInBand --bail",
-    "test-unit": "yarn test",
-    "build:docs": "typedoc && prettier --write docs/**/*.md docs/*.md"
+    "test-unit": "pnpm test",
+    "build:docs": "typedoc && node scripts/fix-links.js && prettier --write docs/**/*.md docs/*.md"
   },
   "devDependencies": {
     "@edge-runtime/jest-environment": "2.0.0",
     "@types/jest": "27.4.1",
     "ts-node": "8.9.1",
     "tsup": "6.1.2",
-    "typedoc": "0.23.10",
-    "typedoc-plugin-markdown": "3.13.4",
-    "typedoc-plugin-mdn-links": "2.0.0",
+    "typedoc": "0.23.24",
+    "typedoc-plugin-markdown": "3.14.0",
+    "typedoc-plugin-mdn-links": "2.0.2",
```

Comment:
Any reason why these deps are updating?

Reply:
Trying to fix the edge docs issue. I can revert now that I have the replacer script